### ISSUE TITLE
Print more information when exception happens in Test Runner

### ISSUE THREAD

**Issue by bsaby** [microsoft/vscode-java-test#676]
### Environment:
```
OS: Windows/Linux
JDK: 8
Java Test Runner Extension: 0.16.0
```

### Repro steps:
1. Git clone https://github.com/bsaby/invisble-project
2. Open the project in VSCode
3. Run all test cases from test explorer

### Result:
Can't run a testng case in the invisible project
![image](https://user-images.githubusercontent.com/22470312/55792198-b3677c80-5af2-11e9-98db-7869e865bf78.png)

**Comment by jdneo** [comment]
## Root cause
the dependency `com.beust.jcommander` is missing in the classpath

## Workaround
https://github.com/cbeust/testng/issues/1932#issuecomment-432512257

## Action to Take
The Test Runner should print more information to help user/dev to diagnostic.

**Comment by jdneo** [comment]
Fixed in #711

**Comment by bsaby** [comment]
Now: 
![image](https://user-images.githubusercontent.com/22470312/58298661-da61df80-7e0e-11e9-853f-0804635caa5e.png)

**Comment by ZhengJun003** [comment]
Test in 0.19.0
No test result show in status bar and no notification prompt out
So reopen this bug

![图片](https://user-images.githubusercontent.com/41051257/64596246-1fbe1300-d3e6-11e9-838a-c32760bb5528.png)

**Comment by jdneo** [comment]
Should be fixed with #806
